src/app/components/ScheduleSummary/ScheduleSummary.css

```diff
@@ -0,0 +1,49 @@
+.schedule-summary-week {
+    display: flex;
+    flex-direction: row;
+    position: inherit;
+}
+.schedule-summary-day {
+    width: 20px;
+    height: 20px;
+    padding-top: 1px;
+    padding-bottom: 1px;
+    padding-left: 1px;
+    padding-right: 1px;
+    text-align: center;
+    -moz-border-radius:50%;
+    -webkit-border-radius: 50%;   
+    border-radius: 50%;
+    background-color: white;
+    border: 1px solid;
+    margin-right: 2px;
+}
+
+.empty-day {
```

Comment:
If you wanted to make sure that these styles don't conflict (i.e. if someone else created a class called .empty-day etc).  What you can do is add the parent class as part of the selector:

```css

.schedule-summary-week .schedule-summary-day {
 //  ... default css for day
}

.schedule-summary-week .empty-day {
 //  css for empty
}

... etc

```

This makes sure that the style is applied when inside of your component and not overridden / overriding someone else's style

Reply:
👌 on the css though, very nice